exercises/saddle-points/canonical-data.json

```diff
@@ -0,0 +1,87 @@
+{
+  "exercise": "saddle-points",
+  "version": "1.0.0",
+  "comments": [
+    "Matrix rows and columns are 0-indexed."
+  ],
+  "cases": [
+    {
+      "description": "Can identify single saddle point",
+      "comments": [
+        "This is the README example."
+      ],
+      "property": "saddlePoints",
+      "input": [
+        [9, 8, 7],
+        [5, 3, 2],
+        [6, 6, 7]
+      ],
+      "expected": [
+        {
+          "row": 1,
+          "column": 0
+        }
+      ]
+    },
+    {
+      "description": "Can identify that empty matrix has no saddle points",
+      "property": "saddlePoints",
+      "input": [
+        []
+      ],
+      "expected": []
```

Comment:
Nice spot, my bad. On reflection, it would seem that this is a case where it makes sense to update the description; I'll push a commit with that change this weekend :o)